spec/model_spec.js

```diff
@@ -2143,9 +2143,10 @@ describe('Model', function () {
       });
 
       it('does not undo changes to owned associations specified in the exception option as an array', function() {
```

Comment:
@stallings Originally the `except` option was designed to only avoid recursively calling `undoChanges` on associated objects. This change makes it work with attributes as well. I think this makes sense, but please update the documentation to reflect that.